pkg/csi/service/node.go

```diff
@@ -473,12 +474,17 @@ func (driver *vsphereCSIDriver) NodeUnpublishVolume(
 	}
 
 	// Figure out if the target path is a file or block volume.
-	isFileMount, _ := common.IsFileVolumeMount(ctx, target, mnts)
+	isFileMount, err := common.IsFileVolumeMount(ctx, target, mnts)
+	if err != nil {
+		return nil, logger.LogNewErrorCodef(log, codes.Internal,
+			"could not find out if file volume mounted: %q", err.Error())
+	}
 	isPublished := true
 	if !isFileMount {
 		isPublished, err = isBlockVolumePublished(ctx, volID, target)
 		if err != nil {
-			return nil, err
+			return nil, logger.LogNewErrorCodef(log, codes.Internal,
```

Comment:
The err returned from this function already has the error code set in it. We can skip this change.